LayoutTests/imported/w3c/web-platform-tests/css/css-tables/parsing/border-spacing-computed-expected.txt

```diff
@@ -1,6 +1,6 @@
 
 PASS Property border-spacing value '10px 20px'
-PASS Property border-spacing value '0'
+FAIL Property border-spacing value '0' assert_equals: expected "0px 0px" but got "0px"
```

Comment:
We should probably re-import the WPT before doing this, it's kinda strange that this appears as a regression

Reply:
Oh it's marked as failing here: https://searchfox.org/wubkat/rev/3c50bf12bcc7cc7cb47c8b3a8d238e5f433dcf35/LayoutTests/TestExpectations#6641 , you should remove that expectation in this patch now that you've made it pass

Reply:
You've also progressed the test above, so you can remove that as well: https://searchfox.org/wubkat/rev/3c50bf12bcc7cc7cb47c8b3a8d238e5f433dcf35/LayoutTests/TestExpectations#6640